### ISSUE TITLE
Problem in creating new Flutter project with Android Studio

### ISSUE THREAD

**Issue by msndvlpr** [flutter/flutter#26820]
I want to create a sample Flutter project with Android Studio but the problem is although I installed Flutter and Flutter language plugin in Android Studio 3.2.1 
the menu is missing in File > New.

I also downloaded the Flutter SDK and configured environment variables. 

Following is my flutter doctor result: 
```
Doctor summary (to see all details, run flutter doctor -v):
Flutter (Channel stable, v1.0.0, on Microsoft Windows [Version 6.1.7601], locale en-US)
Android toolchain - develop for Android devices (Android SDK 28.0.3)
Android Studio (version 2.3)
Flutter plugin version 12.1 - the recommended minimum version is 16.0.0
Android Studio (version 3.2)
IntelliJ IDEA Community Edition (version 15)
Flutter plugin not installed; this adds Flutter specific functionality.
Dart plugin not installed; this adds Dart specific functionality.
This install is older than the minimum recommended version of 2017.1.0.
IntelliJ IDEA Ultimate Edition (version 2018.3)
Connected device
    ! No devices available
! Doctor found issues in 2 categories.
```

**Comment by msndvlpr** [comment]
zoechi, This my flutter doctor -v result:
- Flutter (Channel stable, v1.0.0, on Microsoft Windows [Version 6.1.7601], locale en-US)
 . Flutter version 1.0.0 at C:\flutter
 . Framework revision 5391447fae (7 weeks ago), 2018-11-29 19:41:26 -0800
 . Engine revision 7375a0f414
 . Dart version 2.1.0 (build 2.1.0-dev.9.4 f9ebf21297)

- Android toolchain - develop for Android devices (Android SDK 28.0.3)
 . Android SDK at C:\sdk
 . Android NDK location not configured (optional; useful for native profiling support)
 . Platform android-28, build-tools 28.0.3
 . ANDROID_HOME = C:\sdk
 . Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
 . Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b06)
 . All Android licenses accepted.

- Android Studio (version 2.3)
  . Android Studio at C:\Program Files\Android\Android Studio v2
  . Flutter plugin version 12.1 - the recommended minimum version is 16.0.0
  . Dart plugin version 162.2924
  . Java version OpenJDK Runtime Environment (build 1.8.0_112-release-b06)

- Android Studio (version 3.2)
  . Android Studio at C:\Program Files\Android\Android Studio
  . Flutter plugin version 31.3.1
  . Dart plugin version 181.5656
  . Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b06)

+ IntelliJ IDEA Community Edition (version 15)
 . IntelliJ at C:\Program Files (x86)\JetBrains\IntelliJ IDEA Community Edition 15.0.6
 . Flutter plugin not installed; this adds Flutter specific functionality.
 . Dart plugin not installed; this adds Dart specific functionality.
 . For information about installing plugins, see
 .  https://flutter.io/intellij-setup/#installing-the-plugins
 . This install is older than the minimum recommended version of 2017.1.0.

-IntelliJ IDEA Ultimate Edition (version 2018.3)
   .  IntelliJ at C:\Program Files\JetBrains\IntelliJ IDEA 2018.3.2
   .  Flutter plugin version 31.3.4
   .  Dart plugin version 183.4886.3

[!] Connected device
    ! No devices available

! Doctor found issues in 2 categories.

**Comment by zoechi** [comment]
Also asked on https://stackoverflow.com/questions/54274963/problem-in-creating-new-flutter-project-with-android-studio which is the right place.

So I'm going to close here.
Feel free to create a new issue if it turns out to be a Flutter problem.

**Comment by msndvlpr** [comment]
Why did you close the issue? That's me too on stackoverflow! Isn't this place the official place to say problems about Flutter? Are stackoverflow.com and github.com  the same? You would better to give a solution for mentioned issue instead! @zoechi

**Comment by zoechi** [comment]
GitHub is for Flutter issues and feature requests.

Flutter isn't responsible for your entire system setup. 
Please try to figure out the root cause with support from the community and if it turns out to be a Flutter problem, create a GitHub issue.

There is no point in investigating in parallel on multiple locations.

**Comment by msndvlpr** [comment]
I understand what is Github community for and how should an issue be here. You're right Flutter isn't responsible for my system setup but is responsible to its published docs in flutter.io which hasn't tell anything about such problem and also to Flutter plugin being completely installed on android Studio too, as both of them belong to Google. @zoechi 
Thanks anyhow.

**Comment by faridl29** [comment]
Try enabling the plugins Android APK Support & Android NDK Support.
